front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -469,12 +509,13 @@ class PBExpenses extends PureComponent<Props & InjectedIntlProps, State> {
                 />
 
                 <SubmitExpensesButton
-                  onClick={this.handleSubmitExpensesOnClick}
+                  onClick={handleSubmitExpensesOnClick}
                   bgColor={colors.adminTextColor}
                   disabled={
                     validationStatus === 'validationSuccess' ||
                     budgetExceedsLimit ||
-                    spentBudget === 0
+                    spentBudget === 0 ||
```

Comment:
Could do that, although it just moves the complexity (quite) a few lines up 😬 I don't see the value of doing that atm. It would make things more confusing. Given how big the component is, you'd have to go up quite a bit if you want to change one of the conditions and might lose the context?

Reply:
Could do that, although it just moves the complexity (quite) a few lines up 😬 Wouldn't that make things more confusing?Given how big the component is, you'd have to go up quite a bit if you want to change one of the conditions and might lose the context?

Reply:
Don't have a really strong opinion here, I'm fine with leaving it as is.